eslint.config.js

```diff
@@ -16,4 +16,10 @@ export default [
       'tailwindcss/no-custom-classname': ['error', { callees: ['cn', 'cva'] }],
     },
   },
+  {
+    name: 'custom:no-unused-vars',
+    rules: {
+      'no-unused-vars': 'off',
```

Comment:
question: why?

todo: start a discussion with a team about removing an eslint rule or bring it back. If we remove it now, I don't think we'll remember to put it back afterwards

Reply:
I agree that this rule shouldn't be in here.

Reply:
how often do you see it with the rule enabled?

Reply:
It happened for some files in veil

Reply:
are you bringing the rule back? Maybe disable eslint for some specific lines?